src/ocrd_froc/processor.py

```diff
@@ -113,63 +138,9 @@ def _process_segment(self, segment, image):
                                           method=ocr_method,
                                           classification_result=result,
                                           fast_cocr=fast_cocr,
-                                          adaptive_treshold=adaptive_threshold)
+                                          adaptive_threshold=adaptive_threshold)
```

Comment:
wow!

Reply:
Indeed. No idea how this slipped by.

Reply:
same old: what you did not test, did not happen ;-)